docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -330,9 +214,7 @@ Config:
 role_attribute_path = role
 ```
 
-**Advanced example:**
-
-In the following example user will get `Admin` as role when authenticating since it has a role `admin`. If a user has a role `editor` it will get `Editor` as role, otherwise `Viewer`.
+In the following more complex example, the user is granted the `Admin` role because they have a role `admin`. If a user has a role `editor`, `Editor` role will be granted, otherwise `Viewer`.
```

Comment:
@IevaVasiljeva , I'm unsure of what this means. Do we mean to say: "...the user is granted the system administrator role because they have the `admin` role assignment. If the user has the `editor` role, they are granted the editor role in the system. Otherwise, the user is granted the `Viewer` role."